protocols/ping/Cargo.toml

```
@@ -21,6 +21,7 @@ libp2p-identity = { workspace = true }
 log = "0.4.19"
 rand = "0.8"
 void = "1.0"
+tracing = "0.1.37"
```

```suggestion
tracing = "0.1.37"
void = "1.0"
```

Preference of these to be in alphabetical order.